src/libcrun/cgroup.c

```diff
@@ -44,11 +44,11 @@ static const cgroups_subsystem_t cgroups_subsystems[] = {
   "devices",
   "pids",
   "memory",
-  "net_cls,net_prio",
+  "net_cls",
+  "net_prio",
   "freezer",
   "blkio",
   "hugetlb",
-  "cpu,cpuacct",
```

Comment:
Is this intentional?

Reply:
Yes, met the same error if I just only fix the issue of `net_cls,net_prio`.

Reply:
do we need to add `cpuacct` to the list?

Reply:
Yes.
I should be careful to del this line. I had seen `cpu` in line 43, so I thought `cpuacct` may be in the array, I should check it.